examples/eks_cluster_webshop/tools.tf

```diff
@@ -0,0 +1,74 @@
+resource "helm_release" "kube_prometheus_stack" {
+  name             = "prom-stack"
+  repository       = "https://prometheus-community.github.io/helm-charts"
+  chart            = "kube-prometheus-stack"
+  namespace        = "tools"
+  create_namespace = true
+  cleanup_on_fail  = true
+  wait             = true
+  version          = "35.2.0"
+
+  values = [
+    templatefile("./helm-values/kube-prometheus-stack.yaml.tpl", { grafana_host = var.grafana_host, grafana_password = random_password.grafana_admin.result })
+  ]
+
+  depends_on = [helm_release.nginx]
+}
+
+resource "random_password" "grafana_admin" {
+  length = 5
+}
+
+resource "helm_release" "loki" {
+  name             = "loki"
+  repository       = "https://grafana.github.io/helm-charts"
+  chart            = "loki-simple-scalable"
+  namespace        = "tools"
+  create_namespace = true
+  cleanup_on_fail  = true
+  wait             = true
+  version          = "1.0.0"
```

Comment:
I'm using this helm chart https://github.com/grafana/helm-charts/blob/main/charts/loki-simple-scalable/values.yaml#L442 in it PVC is enabled by default. But I've changed storage class to use EBS CSI Driver instead of default StorageClass

Reply:
Cool, thanks!